src/Files.App/Views/Pages/PropertiesGeneral.xaml

```diff
@@ -441,7 +441,7 @@
 				x:Name="PropertiesDriveCapacity"
 				Grid.Row="2"
 				Grid.Column="1"
-				Padding="28,0,0,0"
+				Padding="25,0,0,0"
```

Comment:
This was very specific, is there any reason it was changed?

Reply:
As explained above, there is a gap of 3 pixels compared to the other 2 texts. The rectangle uses 25 pixels (15 + right margin 10). With this change, texts are aligned vertically. This is visible on the before/after screenshorts.

Reply:
Can you send a screenshot showing the issue? There might be a way to solve it while keeping the divisible by four rule.

Reply:
There is already a screenshot above (before/after). For the multiple of 4, we can change the rectangle, or its margin, because it is not a multiple of 4.

Reply:
I don't the issue in the screenshots

Reply:
Here are the same images with a red line to clarify the problem. We clearly see that the 3rd text is not aligned with the 2 others. Looking at the code, it is clear that the gap should be 25 (used by the rectangle) and not 28.

![fixbefore](https://user-images.githubusercontent.com/46631671/196981606-9df7418d-896d-45ad-acd8-e81a6f122a0d.png)
![fixafter](https://user-images.githubusercontent.com/46631671/196981604-0c2e53d4-451e-4407-b125-cc45df6a7747.png)

Reply:
I'd like to do this differently, can you revert this change and I'll follow up with a separate PR?